Readme.md

```diff
@@ -209,7 +209,7 @@ Some CRD schemas do not have explicit validation for fields implicitly validated
 
 ### Usage as a Github Action
 
-Kubeconform is publishes Docker Images to Github's new Container Registry, ghcr.io. These images
+Kubeconform publishes Docker Images to Github's new Container Registry, ghcr.io. These images
```

Comment:
I would use
`Container Registry – ghcr.io.`
or 
`Container Registry: ghcr.io.`
but the comma is not too bad.